zuul.d/tempest_multinode.yaml

```diff
@@ -63,9 +63,28 @@
       - openstack-k8s-operators/repo-setup
       - openstack-k8s-operators/edpm-ansible
 
+- job:
+    name: cifmw-base-multinode-master-tempest
```

Comment:
wouldn't it be the opposite? you have the master-tempest job, and antelope-tempest job inherit from master, and then set the openstack_release to antelope?

Reply:
yeah, you are right.